src/main/java/frc/robot/logging/RobotLogManager.java

```diff
@@ -0,0 +1,123 @@
+package frc.robot.logging;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.apache.logging.log4j.core.config.Configuration;
+import org.apache.logging.log4j.core.config.ConfigurationFactory;
+import org.apache.logging.log4j.core.config.ConfigurationSource;
+import org.apache.logging.log4j.core.config.Configurator;
+import org.apache.logging.log4j.core.config.yaml.YamlConfigurationFactory;
+
+public class RobotLogManager {
+
+  private static boolean initialized = false;
+
+  private static File directory;
+
+  private static String[] filepaths = { //File paths go in this array
+     "/media/sda1/logging/log4j2.yaml",
+    // "/media/sda2/logging/log4j2.yaml",
+      "/media/sda1/logging/log4j2-test.yaml",
+    // "/media/sda2/logging/log4j2-test.yaml",
```

Comment:
Is there a reason to keep these commented out lines?

Reply:
We are not sure what port the usb will be plugged into